torchbenchmark/models/hf_Albert/__init__.py

```diff
@@ -0,0 +1,75 @@
+
+# Generated by gen_hf_generative.py
```

Comment:
nitpick: can we make this comment visually stand out somehow? Otherwise, inevitably, we'll start editing individual models.